src/java.base/share/classes/jdk/internal/vm/annotation/JvmtiMountTransition.java

```diff
@@ -28,11 +28,18 @@
 import java.lang.annotation.*;
 
 /**
- * A method is annotated as "jvmti mount transition" if it starts
- * or ends virtual thread mount state transition (VTMS transition).
+ * A method may be annotated as "jvmti mount transition" to hint
+ * it is desirable to omit it from JVMTI stack traces.
```

Comment:
Might be better to replace both usages of "jvmti mount transition" with JvmtiMountTransition.

"virtual thread mount state transition (VTMS transition)" should probably be "Virtual Thread Mount State (VTMS) transition".

The updated wording is better but I think this still hard to audit to know if you've got the usages in the right place. Maybe we can re-visit it in the future.

Reply:
I believe it has been fixed with the latest updates.